tutorials/monitoring/3-configuring-endpoints/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Monitoring NServiceBus solutions: Configuring endpoints"
+reviewed: 2017-10-10
+summary: Setting up NServiceBus endpoints to be monitored by the Particular Service Platform.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Throughput and processing time"
+  nextUrl: tutorials/monitoring/4-endpoint-performance
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This third lesson guides you through setting up all of the endpoints in your system to be monitored by the Particular Software Platform.
+
+
+## Configure endpoints
+
+In order to be monitored by the Particular Service Platform, each endpoint needs to be configured to send data to three different queues: error, audit, and monitoring. It is recommended to go through this entire process for a single endpoint and ensure that it is working before continuing on to the next endpoint.
```

Comment:
I would put the last sentence in a separate NOTE - I think it's quite important.